sdl_android/src/androidTest/java/com/smartdevicelink/transport/RegisteredAppTests.java

```diff
@@ -18,47 +18,44 @@
 
     public void testHandleMessage() {
 
-        // Run Test in Main Thread
-        new Handler(Looper.getMainLooper()).post(new Runnable() {
-            @Override
-            public void run() {
+        if (Looper.myLooper() == null)
+        {
```

Comment:
Try to stick to the style of the project and leave the bracket on the same line as the if statement